js/ui/grid_core/ui.grid_core.sorting_mixin.js

```diff
@@ -24,7 +24,7 @@ export default {
             rootElement.find('.' + SORT_CLASS).remove();
             !$indicatorsContainer.children().length && $indicatorsContainer.remove();
 
-            const isSortingAllowed = (sortingMode === 'single' || sortingMode === 'multiple') && column.allowSorting;
+            const isSortingAllowed = sortingMode !== 'none' && column.allowSorting;
```

Comment:
Need to change it, cause in tests there's no default value for `sortingMode`. It should be `'single'`,  but in non-integration tests it's `undefined`